projects/cds/src/services/cds-endpoints.service.spec.ts

```diff
@@ -128,7 +130,7 @@ describe('CdsEndpointsService', () => {
           {
             strategyId: STRATEGY_ID,
           },
-          STRATEGY_REQUEST
+          STRATEGY_REQUEST.queryParams
```

Comment:
this file as well needs to be in lowercase (if the complex object is needed)

Reply:
Done